src/server/sample_data/routes/lib/create_index_name.js

```diff
@@ -17,6 +17,9 @@
  * under the License.
  */
 
-export function createIndexName(server, sampleDataSetId) {
-  return `kibana_sample_data_${sampleDataSetId}`;
+export function createIndexName(sampleDataSetId, dataIndexId) {
+  if (sampleDataSetId === dataIndexId) {
+    return `kibana_sample_data_${sampleDataSetId}`;
+  }
+  return `kibana_sample_data_${sampleDataSetId}_${dataIndexId}`;
```

Comment:
Just want to double-check that these ids are stable (e.g. sampleDataSetId and dataIndexId), because otherwise, updates to sample data might not properly clean out previous indices.

Reply:
yes, the ids should not change

Reply:
OK. And for the current sample datasets, `if (sampleDataSetId === dataIndexId) {` will be true? So, this should still generate the same index name for the current data sets?

Reply:
Actually, might want to slap a comment on there about that, as it's not very obvious that changing that if statement will break the two current datasets.